packages/core/src/local-pin-api.ts

```diff
@@ -9,12 +9,11 @@ import { Repository } from './state-management/repository'
 export class LocalPinApi implements PinApi {
   constructor(
     private readonly repository: Repository,
-    private readonly loadStream: (streamId: StreamID) => Promise<RunningStateLike>,
     private readonly logger: DiagnosticsLogger
   ) {}
 
   async add(streamId: StreamID): Promise<void> {
-    const state$ = await this.loadStream(streamId)
+    const state$ = await this.repository.load(streamId, {})
```

Comment:
let's explicitly pass `{sync: SyncOptions.PREFER_CACHE}` like we used to do in `loaderWithSyncSet` at the higher level